lib/spreewald/web_steps.rb

```diff
@@ -662,18 +662,23 @@
   end
 end.overridable
 
-# Tests that an input or button with the given label is disabled.
+# Tests that an input, button or checkbox with the given label is disabled.
 Then /^the "([^\"]*)" (field|button|checkbox) should( not)? be disabled$/ do |label, kind, negate|
-  if kind == 'field' || kind == 'checkbox'
-    if Capybara::VERSION < "2.1"
-      element = find_field(label, :disabled => !negate)
+  if Capybara::VERSION < "2.1"
```

Comment:
I know you didn't write this code and only changed it - so this is just a reminder. You should be aware that comparing version strings will fail in certain conditions:
```
irb(main):001:0> "2.2" > "2.14"
=> true
```
Therefore we should always cast them to Versions before comparing:
```
irb(main):002:0> Gem::Version.new("2.2") > Gem::Version.new("2.14")
=> false
```
This should have been corrected already but unfortunately the commit still waits to be merged. You may correct it or just leave it as is until the other commit will resolve this.